cwms-data-api/src/main/java/cwms/cda/data/dto/stream/StreamLocation.java

```diff
@@ -66,22 +64,14 @@ private StreamLocation(Builder builder) {
 
     @Override
     public void validate() throws FieldException {
```

Comment:
See change: https://github.com/USACE/cwms-data-api/pull/758/files#diff-56e8053f0501c3bbafc9a273f6aec570244519d33e86897692d23168adf6676aR106 for better validate handling (switch to the `validateInternal` method)

Reply:
Removed validate from streamDTOs and replaced with required annotations